igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -74,8 +76,30 @@ class BuildController {
         if (buildArtifactFilter) {
             build.artifacts = buildArtifactFilter.filterArtifacts(build.artifacts)
         }
+        return build;
+    }
 
-        return build
+    @RequestMapping(value = '/builds/status/{buildNumber}/{master:.+}/**')
```

Comment:
What does the `**` correspond to here?

Reply:
The current pattern in the `BuildController` is to append the Jenkins job name (slashes and all) to the end of the URL, so this (and the `def job` line below) grab everything at the end and consider it to be the job.

I'm not sure why we don't encode the job name and consider it as a single query token (or even make it a parameter) but wanted to be consistent with the other endpoints.  It means you have to send the job name unencoded, which I learned [here](https://github.com/spinnaker/echo/pull/421).